src/components/molecules/OakHandDrawnHR/OakHandDrawnHR.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import styled from "styled-components";
+
+import { OakFlex } from "../../atoms";
+import {
+  InternalStyledSvg,
+  InternalStyledSvgProps,
+} from "../../atoms/InternalStyledSvg";
+
+import { SizeStyleProps } from "@/styles/utils/sizeStyle";
+import { SpacingStyleProps } from "@/styles/utils/spacingStyle";
+
+const StyledOakFlex = styled(OakFlex)``;
+
+export type OakHandDrawnHRProps = {
+  fill?: InternalStyledSvgProps["$fill"];
```

Comment:
I think we could be more selective with the exposed props here. Does the user really need access to strokeWidth ? And do the color controls really need to be independent for stroke and fill ? Could you reduce this list to the bare essentials and hard code the logic for the rest ?

Reply:
agreed